Array & Hash/three_sum.cpp

```diff
@@ -0,0 +1,36 @@
+class Solution {
```

Comment:
Can you please add a question link and Time Complexity for this solution.

Reply:
Sure